expertsystem/topology.py

```diff
@@ -41,22 +41,23 @@ def __eq__(self, other: object) -> bool:
         raise NotImplementedError
 
 
-class StateTransitionGraph:
-    """Graph class that contains edges and nodes.
+class Topology:
+    """Directed graph for a state transition.
```

Comment:
The thing is that this is not a normal graph of graph theory (CS). I think some comment explaining the difference would be good (maybe even a reference to feynman graphs)

Reply:
5bca7a79be63fe5f18b2c10e151dce66b3f6dd62